pycloudflare/utils.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+
+from pycloudflare.exceptions import SSLUnavailable
+from pycloudflare.services import HTTPServiceError
+
+
+errors_mapping = {
+    1001: SSLUnavailable,
+}
```

Comment:
Taking into account that CF have their error codes scoped to the certain API (or collection of APIs), it sounds logical to have that separation on our side as well.  I mean a mechanism (decorator) of error code retrieval and code comparing, applied to certain API, will take the collection of error codes specific to that API.  Do you have any other workaround for that?